amgut/db/patches/0010.sql

```diff
@@ -0,0 +1,22 @@
+CREATE TABLE ag.external_survey ( 
+	external_survey_id   bigserial  NOT NULL,
+	external_survey      varchar  NOT NULL,
+	external_survey_description varchar  ,
+	CONSTRAINT pk_external_surveys PRIMARY KEY ( external_survey_id )
+ );
+
+CREATE TABLE ag.external_survey_answers ( 
+	survey_id            varchar  NOT NULL,
+	external_survey_id   bigint  NOT NULL,
+	pulldown_date        date  NOT NULL,
+	answers              json  NOT NULL,
+	CONSTRAINT pk_third_party_surveys PRIMARY KEY ( survey_id )
```

Comment:
The first combined key idea is probably best. I didn't combine the two tables because it makes it harder to add/remove/query for what external surveys are available to ag users. I originally had it as an enum for external_survey_id to control what surveys there were, but it got cumbersome for adding and querying survey availability.